### PR TITLE
Add Update and Troubleshoot to Mac install guide

### PR DESCRIPTION
**Pull request by albertocalderari**

### CODE REVIEW

**Comment by albertocalderari** [comment]
@team-pct Small improvement to the documentation

**Comment by kiriles90** [comment]
@albertocalderari What is the empty .Rhistory file?

File: README.md [hunk 1]
```diff
@@ -49,6 +49,10 @@ Easily install Popcorn Time via _[Homebrew](https://brew.sh) ([Cask](https://git
   * Or **latest dev build (for testers)**:  
   `brew tap popcorn-official/popcorn-desktop https://github.com/popcorn-official/popcorn-desktop.git`  
   `brew install --cask popcorn-time-beta`
+  * **Update from zip file**:
+  Unpack the content of the zip file. This should leave you with a file called `update.tar`. Unpacking the .tar file you should get you a folder called `Content`. Navigate to your `Appplication` folder, right click on Popcotn-Time and press `Show Package Content`. Replace te `Content` folder in the app with the content folder from the `update.zip` file.
+  * **Troubleshoot**:
+  If you are running little snitch, try disabling the network monitor. Start downloading a movie and check that the artwork is dowloaded. If the artwork is downloaded the movie should start downloading in a minute or so.
```
Comment (kiriles90) [inline README.md:55]: Can we remove the `**Troubleshoot**` section? Its a good advice for people that have issues with connections its a bit too specific plus then we'd have to add all the other issues people might have on different OSes and the README will turn into a troubleshooting guide with 20 times the size.

We can however make a wiki page specifically for troubleshooting or troubleshooting on macOS if you want to give a go at that.

Comment (albertocalderari) [reply README.md:55]: Ok that sounds good, I was debated a bit too tbhwy

**Pull request closed without merging**